command/agent/agent.go

```diff
@@ -326,6 +326,23 @@ func convertServerConfig(agentConfig *Config) (*nomad.Config, error) {
 		}
 	}
 
+	jobMaxPriority := int(structs.JobDefaultMaxPriority)
```

Comment:
Isn't this already an int?

```suggestion
	jobMaxPriority := structs.JobDefaultMaxPriority
```